package.json

```diff
@@ -85,7 +85,7 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
-    "aws-sdk": "^2.585.0",
+    "aws-sdk": "^2.666.0",
```

Comment:
If this was changed for a reason, I don't think it's going to have the expected impact. The two modified packages (aws-client and ingest) import `aws-sdk` on their own and do not use the version specified here.

Reply:
Yep, sorry. I was experiencing some strange behavior with the AWS SDK that I thought might be fixed by upgrading. It turned out not to be the case, but I forgot to revert this before committing. I'll revert.